spread.yaml

```diff
@@ -499,7 +497,7 @@ debug-each: |
         fi
 
         echo '# journal messages for snapd'
-        get_journalctl_log -u snapd
+        "$TESTSTOOLS"/journal-state get-journalctl-log -u snapd
```

Comment:
Leftover old name? This should be `get-log`, right?

Reply:
yes, forgot to update that one, thanks!